LibreSSL does not support SSL_set_quic_early_data_enabled.

Index: src/tls/babassl/xqc_ssl_if.c
--- src/tls/babassl/xqc_ssl_if.c.orig
+++ src/tls/babassl/xqc_ssl_if.c
@@ -44,7 +44,9 @@ xqc_ssl_is_early_data_enabled(SSL_SESSION *session)
 void
 xqc_ssl_enable_max_early_data(SSL *ssl)
 {
+#if 0
     SSL_set_quic_early_data_enabled(ssl, 1);
+#endif
 }
 
 
